docs/getting-started/user-guides/decrediton-setup.md

```diff
@@ -6,13 +6,13 @@ Last updated for v1.0.0.
 
 `Decrediton` is a graphical user interface for `dcrwallet`. When this application launches, it automatically starts its own instance of `dcrd` and `dcrwallet` in the background - it will not open if there is already a running instance of `dcrd`.
 
-NOTE: At any point in the use of Decrediton, it may become unresponsive or get stuck on a loading screen. This can usually be fixed with a restart of the application.
+NOTE: If at any point the program becomes unresponsive or gets stuck on a loading screen, this can usually be fixed with a restart of the application.
```

Comment:
This change is good, but can you mention it in the next commit message.  Shouldn't have changes that aren't referenced at least somewhat in a commit message.